dubbo-common/src/main/java/org/apache/dubbo/common/utils/PojoUtils.java

```diff
@@ -515,7 +515,7 @@ private static Object newInstance(Class<?> cls) {
         } catch (Throwable t) {
             try {
                 Constructor<?>[] constructors = cls.getDeclaredConstructors();
-                if (constructors == null || constructors.length == 0) {
+                if (constructors.length == 0) {

```

Comment:
Please add a comment here, why the null check has been removed.

Reply:
Rogar~

Reply:
@satansk Agree! Line 524 should be removed.

Reply:
Yes, this is better.